test/Makefile

```diff
@@ -13,13 +13,7 @@ perf:
 
 clean:
 	@$(MAKE) -C perf $@
-	-rm -f libccalltest.$(SHLIB_EXT) ccalltest
 
-.PHONY: $(TESTS) perf clean libccalltest
+.PHONY: $(TESTS) perf clean
 
-all ccall libccalltest: libccalltest.$(SHLIB_EXT)
-libccalltest.$(SHLIB_EXT): ccalltest.c
-	@$(call PRINT_CC, $(CC) $(CFLAGS) $(DEBUGFLAGS) -O3 $< $(fPIC) -shared -o $@ $(LDFLAGS) -DCC="$(CC)")
-
-ccalltest: ccalltest.c libccalltest.$(SHLIB_EXT)
-	@$(call PRINT_CC, $(CC) $(CFLAGS) $(DEBUGFLAGS) -O3 $< -o $@ $(LDFLAGS) -DCC="$(CC)")
+all:
```

Comment:
i'm not exactly sure what gmake will do with this line (`all:`), probably best just to delete it